app/src/main/java/org/jboss/hal/client/configuration/subsystem/datasource/wizard/NamesStep.java

```diff
@@ -57,6 +57,12 @@
                     wizard().getContext().dataSource.update(form.getModel());
                 })
                 .build();
+
```

Comment:
I think it's worth moving this into its own class under `org.jboss.hal.ballroom.form`. There are other resources / form items which need the same validation.

Reply:
Sorry for the delay, you mean like [this](https://github.com/michpetrov/hal.next/blob/a36a53063b50a8df18db23a23a1ea4ded7aad2f2/ballroom/src/main/java/org/jboss/hal/ballroom/form/PatternValidation.java)?

Reply:
Yes, that's what I had in mind.